src/Microsoft.Azure.WebJobs.Host/TypeUtility.cs

```diff
@@ -98,6 +98,25 @@ internal static Type UnwrapTaskType(Type type)
             return null;
         }
 
+        internal static TAttribute GetResolvedAttribute<TAttribute>(ParameterInfo parameter) where TAttribute : Attribute
+        {
+            var attribute = parameter.GetCustomAttribute<TAttribute>();
+
+            var storageAttribute = attribute as StorageAttribute;
```

Comment:
I'm fine with limiting it, but we still need to address the 3 points above. 

---
In reply to: [107999544](https://github.com/Azure/azure-webjobs-sdk/pull/1084#discussion_r107999544) [](ancestors = 107999544)